### ISSUE TITLE
Spark2 ODP-1.2.2.0 install failing on RHEL9 

### ISSUE THREAD

**Issue by lukefoskey** [clemlabprojects/ambari#95]
Hi team, firstly thank you for keeping Ambari alive!

We're facing the below issue when trying to install a new datanode with spark2-client. It appears /lib/lsb/init-functions is causing issues.

```Traceback (most recent call last):
  File "/var/lib/ambari-agent/cache/stacks/ODP/1.2/services/SPARK2/package/scripts/spark_client.py", line 55, in <module>
    SparkClient().execute()
  File "/usr/lib/ambari-agent/lib/resource_management/libraries/script/script.py", line 350, in execute
    method(env)
  File "/var/lib/ambari-agent/cache/stacks/ODP/1.2/services/SPARK2/package/scripts/spark_client.py", line 34, in install
    self.install_packages(env)
  File "/usr/lib/ambari-agent/lib/resource_management/libraries/script/script.py", line 849, in install_packages
    Package(name,
  File "/usr/lib/ambari-agent/lib/resource_management/core/base.py", line 164, in __init__
    self.env.run()
  File "/usr/lib/ambari-agent/lib/resource_management/core/environment.py", line 163, in run
    self.run_action(resource, action)
  File "/usr/lib/ambari-agent/lib/resource_management/core/environment.py", line 127, in run_action
    provider_action()
  File "/usr/lib/ambari-agent/lib/resource_management/core/providers/packaging.py", line 30, in action_install
    self._pkg_manager.install_package(package_name, self.__create_context())
  File "/usr/lib/ambari-agent/lib/ambari_commons/repo_manager/yum_manager.py", line 226, in install_package
    shell.repository_manager_executor(cmd, self.properties, context)
  File "/usr/lib/ambari-agent/lib/ambari_commons/shell.py", line 756, in repository_manager_executor
    raise RuntimeError(message)
RuntimeError: Failed to execute command '/usr/bin/yum -y install spark2_1_2_2_0_138', exited with code '1', message: 'Repository ambari-clemlabs-2.7.9.0.0-110 is listed more than once in the configuration

Error: 

 Problem: conflicting requests

  - nothing provides /lib/lsb/init-functions needed by spark2_1_2_2_0_138-2.4.8.1.2.2.0-138.el9.noarch from ODP-1.2-repo-1```

**Comment by lucasbak** [comment]
Hi @lukefoskey,

Thank you for the report.
Indeed there is a dependency missing on rhel9, we will fix it in the next version of ODP.

Regards 

Clemlab team

**Comment by lucasbak** [comment]
Fixed in ODP 1.2.4.0

Closing the issue after 30 days of inactivity
